packages/grafana-data/src/field/fieldState.ts

```diff
@@ -30,39 +30,39 @@ export function getFrameDisplayTitle(frame: DataFrame, index?: number) {
   return `Series (${index})`;
 }
 
-export function getFieldTitle(field: Field, frame?: DataFrame, allFrames?: DataFrame[]): string {
-  const existingTitle = field.state?.title;
+export function getFieldDisplayName(field: Field, frame?: DataFrame, allFrames?: DataFrame[]): string {
+  const existingTitle = field.state?.displayName;
 
   if (existingTitle) {
     return existingTitle;
   }
 
-  const title = calculateFieldTitle(field, frame, allFrames);
+  const displayName = calculateFieldDisplayName(field, frame, allFrames);
   field.state = {
     ...field.state,
-    title,
+    displayName,
   };
 
-  return title;
+  return displayName;
 }
 
 /**
- * Get an appropriate display title. If the 'title' is set, use that
+ * Get an appropriate display name. If the 'title' is set, use that
  */
-function calculateFieldTitle(field: Field, frame?: DataFrame, allFrames?: DataFrame[]): string {
+function calculateFieldDisplayName(field: Field, frame?: DataFrame, allFrames?: DataFrame[]): string {
   const hasConfigTitle = field.config?.title && field.config?.title.length;
 
-  let title = hasConfigTitle ? field.config!.title! : field.name;
+  let displayName = hasConfigTitle ? field.config!.title! : field.name;
```

Comment:
just so I understand, renaming `field.config.title` to `field.config.displayName` would mean migrations right? IMO we should do this now and not later :)

Reply:
yes lets do this now -- not widely used yet, and better if we start more consistent